### PR TITLE
Bumping-up miniVite to version 1.1

### PR DESCRIPTION
**Pull request by rvinaybharadwaj**

### CODE REVIEW

File: var/spack/repos/builtin/packages/minivite/package.py [hunk 1]
```diff
@@ -62,4 +62,7 @@ def build_targets(self):
 
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
-        install('miniVite', prefix.bin)
+        if (self.version == Version('1.0')):
+            install('dspl', prefix.bin)
+        elif (self.version == Version('1.1')):
```
Comment (junghans) [inline var/spack/repos/builtin/packages/minivite/package.py]: I would make that `>=`.

Comment (rvinaybharadwaj) [reply var/spack/repos/builtin/packages/minivite/package.py]: Making it >= will always default to Version('1.0') install statement. They have changed the name of the target in the makefile, which results in an error if the checked out version is different.

Comment (rvinaybharadwaj) [reply var/spack/repos/builtin/packages/minivite/package.py]: I'll just arrange it in descending order, so the latest version is installed first.

Comment (junghans) [reply var/spack/repos/builtin/packages/minivite/package.py]: Sorry, that is what I meant!

**Comment by rvinaybharadwaj** [comment]
> Is this binary file in this commit intentional?

Oops, No. That one slipped through my .gitignore by accident.

**Pull request merged**